src/lib/Repository/UiTranslationService.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Repository;
+
+use eZ\Publish\API\Repository\LanguageService;
+use eZ\Publish\API\Repository\Values\Content\VersionInfo;
+
+/**
+ * Service for loading translations.
+ */
+class UiTranslationService
```

Comment:
I'm a bit skeptical at a separate service for this. Having language handling inside the `UiUserService` wouldn't make it _too_ complex, would it ?